packages/alpha-app/src/app.js

```diff
@@ -0,0 +1,153 @@
+const express = require("express");
```

Comment:
`const dotenv = require("dotenv")
dotenv.config()`

Is it worth adding this in whilst we're adding in the initalisation

Reply:
Thanks yeah that makes sense to me. I've added the `.env` file and config.